ftva_lab_data/forms.py

```diff
@@ -62,3 +63,10 @@ class Meta:
             "arsc_lto_tape_id": "ARSC LTO Tape ID",
             "hard_drive_barcode_id": "Hard drive barcode ID",
         }
+
+        widgets = {"status": forms.CheckboxSelectMultiple}
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Instead of forcing the order here in the form, I believe it's better to do it in the model:
https://docs.djangoproject.com/en/5.2/ref/models/options/#ordering